components/scream/tests/coupled/dynamics_physics/model_restart/input_baseline.yaml

```diff
@@ -12,41 +12,36 @@ Time Stepping:
   Case Start Date: [2021, 10, 12]    # Year, Month, Day
 
 Initial Conditions:
+  Filename: ${SCREAM_DATA_DIR}/init/homme_physics_ne2np4.nc
   Restart Run: false
   Physics GLL:
-    Filename: ${SCREAM_DATA_DIR}/init/homme_physics_ne2np4.nc
     aero_g_sw: 0.0
     aero_ssa_sw: 0.0
     aero_tau_sw: 0.0
     aero_tau_lw: 0.0
 
-Atmosphere Processes:
-  Number of Entries: 5
+atmosphere_processes:
+  atm_procs_list: (homme,shoc,CldFraction,p3,rrtmgp)
```

Comment:
Nice, does this approach mean that if we wanted to rearrange the order we could just change this entry and nothing else?

Reply:
Yes.

Reply:
Well, actually, we do _not_ allow changing this entry via `atm-config-chg`, since changing this would require a full re-generation of it (b/c we might need to go grab other defaults that are not currently in the XML).

The idea behind atm-config-chg is that once namelist_scream.xml is generated, you no longer need to look at namelist_defaults_scream.xml. So if you need sections from the defaults that are not in the current XML, it is an invalid use.

If you look at the defaults XML, you will notice that this entry has the attribute `locked="true"`, which makes atm-config-chg throw an error if you try to change it.

Atm proc order needs to be defined via compset. At least for now.